processor/test/data/coco-cart.json

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
I have created and then modified a CoCo cart response based on some test data we use. I changed all of the UUID `ids` to the same value just in case.